### PR TITLE
Move time and time accel window to lua

### PR DESCRIPTION
**Pull request by ecraven**
Moved the time and time acceleration display to lua.

time-window

### CODE REVIEW

File: data/pigui/modules/time-window.lua [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	local year, month_num, day, hour, minute, second = Game.GetDateTime()
+	local month = lc[months[month_num]]
+	local date = string.format("%04i %s %i - %02i:%02i:%02i", year, month, day, hour, minute, second)
+	local current = Game.GetTimeAcceleration()
+	local requested = Game.GetRequestedTimeAcceleration()
+	function accelButton(name, tooltip)
+		local color = bg_color
+		if requested == name and current ~= name then
+			color = colors.white
+			tooltip = 'Requested ' .. tooltip
```
Comment (laarmen) [inline data/pigui/modules/time-window.lua]: I don't know if we had it before, but what about the localization?

File: data/pigui/modules/time-window.lua [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		end
+		local res = false
+		if ui.coloredSelectedIconButton(icons['time_accel_' .. name], button_size, current == name, frame_padding, color, fg_color, tooltip) then
+			Game.SetTimeAcceleration(name)
+			res = true
+		end
+		ui.sameLine()
+		return res
```
Comment (laarmen) [inline data/pigui/modules/time-window.lua]: I don't see this return value used anywhere.

File: data/pigui/modules/time-window.lua (hunk 1 above)
Comment (ecraven) [inline data/pigui/modules/time-window.lua]: Thanks for the catch, I supposed this was going away anyway, and didn't look at it again.
I'll fix it.

File: data/pigui/modules/time-window.lua (hunk 2 above)
Comment (ecraven) [inline data/pigui/modules/time-window.lua]: Ah, this was used in an alternative version, which I removed. I'll delete it ;)

**Comment by ecraven** [comment]
For Shift-F1..F5 to work, we need to merge #4026 first ;)

**Comment by laarmen** [comment]
About the two commits, rebase gone wrong? Please squash them before
@impaktor sees them and goes berserk ;-).
Otherwise, there's another bit of l10n missing, the 10x time labels.

**Pull request merged**